src/routes.js

```diff
@@ -6,306 +7,312 @@
 /**
  * Routes are all defined here
  */
-export default [
-  {
-    path: `/`,
-    component: () => import(`./components/common/NetworkSetter`)
-  },
-  {
-    path: `/networks`,
-    name: `networks`,
-    component: () => import(`./components/network/PageNetworks`),
-    meta: {
-      requiresAuth: false
-    }
-  },
-  {
-    path: `/welcome`,
-    name: `welcome`,
-    components: {
-      session: () => import(`./components/common/TmSessionWelcome`)
+export default (apollo, store) => {
+  return [
+    {
+      path: `/`,
+      beforeEnter: (to, from, next) =>
+        setNetwork({ to, from, next }, apollo, store),
+      component: () => import(`./components/common/NetworkSetter`)
```

Comment:
This component doesn't exist anymore

Reply:
? it does still exist. And still necessary it seems. It is located in `src/components/common`

Reply:
this is useless, need to remove